src/sql/platform/accounts/browser/accountManagement.contribution.ts

```diff
@@ -64,12 +64,12 @@ ExtensionsRegistry.registerExtensionPoint<IAccountContrib | IAccountContrib[]>({
 			const iconClass = id;
 			if (typeof icon === 'string') {
 				const path = resources.joinPath(extension.description.extensionLocation, icon);
-				createCSSRule(`.icon.${iconClass}`, `background-image: ${asCSSUrl(path)}`);
+				createCSSRule(`.codicon.${iconClass}`, `background-image: ${asCSSUrl(path)}`);
 			} else {
 				const light = resources.joinPath(extension.description.extensionLocation, icon.light);
 				const dark = resources.joinPath(extension.description.extensionLocation, icon.dark);
-				createCSSRule(`.icon.${iconClass}`, `background-image: ${asCSSUrl(light)}`);
-				createCSSRule(`.vs-dark .icon.${iconClass}, .hc-black .icon.${iconClass}`, `background-image: ${asCSSUrl(dark)}`);
+				createCSSRule(`.codicon.${iconClass}`, `background-image: ${asCSSUrl(light)}`);
+				createCSSRule(`.vs-dark .icon.${iconClass}, .hc-black .codicon.${iconClass}`, `background-image: ${asCSSUrl(dark)}`);
```

Comment:
Looks like there's an .icon here you missed for .vs-dark

Reply:
Good catch, thanks!